CMakeLists.txt

```diff
@@ -9,17 +9,17 @@
 #
 
 cmake_minimum_required(VERSION 3.10.2)
-set(CGA_PROJECT_NAME ClaraGenomicsAnalysis)
-file(READ "${CMAKE_CURRENT_SOURCE_DIR}/VERSION" CGA_VERSION)
-string(STRIP ${CGA_VERSION} CGA_VERSION)
-project(${CGA_PROJECT_NAME})
+set(GW_PROJECT_NAME ClaraGenomicsAnalysis)
+file(READ "${CMAKE_CURRENT_SOURCE_DIR}/VERSION" GW_VERSION)
+string(STRIP ${GW_VERSION} GW_VERSION)
+project(${GW_PROJECT_NAME})
 
 # Process options.
-option(cga_enable_tests "Build ClaraGenomicsAnalysis unit tests" OFF)
```

Comment:
`ClaraGenomicsAnalysis` appears in this file several times

Reply:
good catch! changed

Reply:
Changed as well